modin/pandas/io.py

```diff
@@ -128,6 +152,158 @@ def read_csv(
+    |                         |                 |                | Non-lazy       |                                  |
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | keep_date_col           |                 |                | Harmful        |                                  |
+    |                         |                 |                | Non-lazy       |                                  |
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | date_parser             |                 |                | Harmful        |                                  |
+    |                         |                 |                | Non-lazy       |                                  |
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | dayfirst                |                 |                | Harmful        |                                  |
+    |                         |                 |                | Non-lazy       |                                  |
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | cache_dates             |                 |                | Harmful        |                                  |
+    |                         |                 |                | Non-lazy       |                                  |
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | iterator                |                 |                | Harmful        |                                  |
+    |                         |                 |                | Non-lazy       |                                  |
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | chunksize               | Harmful         | Harmful        | Harmful        |                                  |
+    |                         |                 |                | Non-lazy       |                                  |
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | compression             |                 |                | Non-lazy       | **OmniSci**:                     |
+    |                         |                 |                |                | `compression` is inferred        |
+    |                         |                 |                |                | automatically and shouldn't be   |
+    |                         |                 |                |                | specified                        |
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
```

Comment:
if it is possible we can "merge" parameters with similar values for all angines in one cell for example

Reply:
Actually, it is possible, so similar cells were merged.

Reply:
Nice, thanks!